src/Popover/components/ContentWrapper.js

```diff
@@ -78,7 +78,6 @@ const StyledOverlay = styled.div`
 
   ${media.largeMobile(css`
     display: none;
-    z-index: ${({ isInsideModal }) => (isInsideModal ? "999" : "599")};
```

Comment:
This should be reverted, right @filipdanisko?

Reply:
@filipdanisko 🙏

Reply:
No need, due to display none present :)